### DISCUSSION TITLE
Custom Geo table for private addresses

### DISCUSSION THREAD

**Discussion by willichan** [GitSquared/edex-ui#1104] [Ideas]
It would be great if I could include longitude/latitude data (CSV?) for private address ranges on our network.  Most of my network traffic is on the corporate WAN, so I only ever have anything display on the globe if I ping something on the internet.

**Comment by GitSquared** [comment]
This seems like a good solution to a niche problem, but I'm curious about your usecase for edex in a corporate setting?
